packages/@aws-cdk/aws-docdb/lib/database-secret.ts

```diff
@@ -48,7 +48,16 @@ export interface DatabaseSecretProps {
  * @resource AWS::SecretsManager::Secret
  */
 export class DatabaseSecret extends Secret {
+
+  /**
+   * the excluded characters for this Secret
+   */
+  readonly excludedCharacters: string;
```

Comment:
Missing `public`?

Reply:
yes! I was just wondering why it works ... but it seams as if public is TS default... but setting it public in the code is better!